archivist/subjects.py

```diff
@@ -90,6 +90,24 @@ def create(
             ),
         )
 
+    def exchange(self, name: str, subject: Subject) -> Subject:
```

Comment:
Naming this `exchange` might suggest that both parties are adding each other (i.e. exchanging phone numbers)

Reply:
import